tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -127,11 +127,11 @@ def buildProject(self):
             os.environ['PATH'] = path + os.pathsep + os.environ['PATH']
 
         buildCommands = self._buildConfig['commands']
-        print('Building project {}...'.format(self._buildConfig['project_dir']))
+        print('Building project {} ...'.format(self._buildConfig['project_dir']))
         for command in buildCommands:
             command = command.format(**self._buildConfig)
             print('====> Executing Command: ' + command)
-            proc = subprocess.Popen(command + ' >> build_output.txt 2>&1', shell=True)
+            proc = subprocess.Popen(command + ' > build.log 2>&1', shell=True)
```

Comment:
Will the single "\>" still work on windows?

Reply:
My thought is we can use a folder with timestamp based name to keep all logs. I will make change for this after BLE release.

Reply:
Harnisha will use separate temp folder for the build in Jenkins, so both build.log and other build artifacts will be reserved.